contribute-docs/how-to/cumulative-docs/badge-placement.md

```diff
@@ -0,0 +1,207 @@
+
+::::{image} ./images/table-entire-row-incorrect.png
+:screenshot:
+:alt:
+::::
+
+#### If the badge is relevant to one cell, add the badge to the cell it applies to [table-cell]
+
+Add the badge to the end of the content in a cell to indicate that it applies to that one cell only.
+
+For example, the [Collect application data](https://www.elastic.co/docs/solutions/observability/apm/collect-application-data#_capabilities) page includes a table that compares functionality across two methods for collecting APM data, and only one of the methods is in technical preview.
+
+::::{image} ./images/table-one-cell-correct.png
+:screenshot:
+:alt:
+::::
+:::
+
+:::{tip}
+If the one cell that the badge applies to is in the first column, consider formatting the content
+using something other than a table (for example, a definition list) to avoid confusion with the
+[previous scenario](#table-row) in which adding the badge to the first column indicates that the
+badge applies to the whole row.
+:::
+
+#### If the badge is relevant to part of a cell, follow the appropriate placement guidelines for the elements used in the cell [table-cell-part]
+
+For example, the [Parse AWS VPC Flow Log](https://www.elastic.co/docs/reference/beats/filebeat/processor-parse-aws-vpc-flow-log) page includes new information relevant to 9.2.0 and later about a setting that already existed before 9.2.0. In this example, the `applies_to` badges should be at the beginning of each list item as described in [the guidelines for lists](#ordered-and-unordered-lists).
+
+::::{image} ./images/table-part-of-cell-correct.png
+:screenshot:
+:alt:
+::::
+:::
+
+% Reference: https://github.com/elastic/kibana/pull/229485/files#r2231856744
+:::{tip}
+If needed, break the contents of the cell into multiple lines using `<br>` to isolate the content you're labeling or consider not using a table to format the related content.
+:::
+
+### Tabs
+
+In the future ([elastic/docs-builder#1436](https://github.com/elastic/docs-builder/issues/1436)), tabs will be able to accept `applies_to` information. Until then, if an `applies_to` badge is relevant to the entire tab item, add the badge to the beginning of the content.
+
+% TO DO: Add example
+% <image>
+
+### Admonitions
+
+In the future ([elastic/docs-builder#1436](https://github.com/elastic/docs-builder/issues/1436)), admonitions will be able to accept `applies_to` information. Until then, if an `applies_to` badge is relevant to the entire admonition, add the badge to the beginning of the content.
+
+::::{image} ./images/admonition-correct.png
+:screenshot:
+:alt:
+::::
+
+### Dropdowns
+
+In the future ([elastic/docs-builder#1436](https://github.com/elastic/docs-builder/issues/1436)), dropdowns will be able to accept `applies_to` information. Until then, if an `applies_to` badge is relevant to the entire dropdown, add the badge to the beginning of the content.
+
+% TO DO: Add example
+% <image>
```

Comment:
these all have functioning syntax now.

Reply:
let the great uncommenting commence

Reply:
Pretty sure I already updated this somewhere 🤔

Reply:
Weird, https://github.com/elastic/docs-builder/pull/2104 is from 2 weeks ago and is live on the docs-builder docs since then